enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -668,7 +669,7 @@ func (r *dockerCommandContainer) exec(ctx context.Context, command *repb.Command
 	// signals for the signals we care about, particularly SIGKILL, and only in
 	// the case where we are expecting a SIGKILL due to the container being
```

Comment:
TIL: you can exit with a code greater than 127

may want to update this comment